proteus/tests/SWFlow/test_SWFlows.py

```diff
@@ -31,10 +31,10 @@ def compare_vs_saved_files(self,name):
         actual = tables.open_file(name+'.h5','r')
         expected_path = 'comparison_files/' + 'comparison_' + name + '_h_t2.csv'
         #write comparison file
-        # write_path = './' + 'comparison_' + name + '_h_t2.csv'
+        # write_path = './comparison_files/' + 'comparison_' + name + '_h_t2.csv'
         # np.array(actual.root.h_t2).tofile(os.path.join(self._scriptdir, write_path),sep=",")
         #
-        np.testing.assert_almost_equal(np.fromfile(os.path.join(self._scriptdir, expected_path),sep=","),np.array(actual.root.h_t2).flatten(),decimal=7)
+        # np.testing.assert_almost_equal(np.fromfile(os.path.join(self._scriptdir, expected_path),sep=","),np.array(actual.root.h_t2).flatten(),decimal=7)
```

Comment:
Did you mean to comment out this assert?

Reply:
Nope, this was an accident. Let me fix it.